samples/go/main.go

```diff
@@ -4,14 +4,66 @@ package main
 // #cgo LDFLAGS: -L/wallet-core/build -L/wallet-core/build/trezor-crypto -lTrustWalletCore -lprotobuf -lTrezorCrypto -lc++ -lm
 // #include <TrustWalletCore/TWHDWallet.h>
 // #include <TrustWalletCore/TWString.h>
+// #include <TrustWalletCore/TWData.h>
+// #include <TrustWalletCore/TWPrivateKey.h>
+// #include <TrustWalletCore/TWPublicKey.h>
 import "C"
 
 import "fmt"
+import "unsafe"
+import "encoding/hex"
+
+// C.TWString -> Go string
+func TWStringGoString(s unsafe.Pointer) string {
+    return C.GoString(C.TWStringUTF8Bytes(s))
+}
+
+// Go string -> C.TWString
+func TWStringCreateWithGoString(s string) unsafe.Pointer {
+    cStr := C.CString(s)
+    defer C.free(unsafe.Pointer(cStr))
+    str := C.TWStringCreateWithUTF8Bytes(cStr)
+    return str
+}
+
+// C.TWData -> Go byte[]
+func TWDataGoBytes(d unsafe.Pointer) []byte {
```

Comment:
Could we add codegen for it?

Reply:
Yes, these two helpers must be native, we have similar helpers for Java/Objc, we can add go binding to codegen later (if we really need it)